### DISCUSSION TITLE
Separate pack for invisible item frames

### DISCUSSION THREAD

**Discussion by IdotIcom** [BedrockTweaks/Files#330] [Ideas]
One for glowing item frame
                     And
One for regular item frame.

**Comment by SquatchHunter** [comment]
this might be better for subpacks and or ingame config
since ideally we would be able to do something like VT does where they "set" an item frame to invis, not all item frames being invis at all times.

**Reply by IdotIcom** [reply]
That would be really nice too.

Hope y'all do it

**Reply by PAPISOP** [reply]
> this might be better for subpacks and or ingame config since ideally we would be able to do something like VT does where they "set" an item frame to invis, not all item frames being invis at all times.

Not too sure if the subpacks is a good idea as in our website we can just have two different packs which is more better (also i dont think DrAv backend will add subpacks to the manifest)

**Comment by yzabeast1** [comment]
@IdotIcom the invis item frames have now been split into normal and glowing frame packs on [becomtweaks](https://becomtweaks.github.io/resource-packs)
